### ISSUE TITLE
[Fabric] 'rndatetimepicker/ComponentDescriptors.h' file not found

### ISSUE THREAD

**Issue by sontruongna** [react-native-datetimepicker/datetimepicker#665]
# Bug report

## Summary
Build Failded with error "'rndatetimepicker/ComponentDescriptors.h' file not found" 

## Reproducible sample code

<img width="868" alt="Screen Shot 2022-09-27 at 23 28 59" src="https://user-images.githubusercontent.com/93734269/192585860-8ea81f2a-f9ea-45ac-9370-b5a7eec117ed.png">

## Steps to reproduce

1. npx react-native init example
2. yarn add @react-native-community/datetimepicker
3.  cd "example/ios" && RCT_NEW_ARCH_ENABLED=1 npx pod-install && cd -

## Environment info

`npx react-native info` output:

```bash
System:
    OS: macOS 12.6
    CPU: (8) arm64 Apple M1
    Memory: 100.84 MB / 16.00 GB
    Shell: 5.8.1 - /bin/zsh
  Binaries:
    Node: 18.7.0 - /opt/homebrew/bin/node
    Yarn: 1.22.11 - /opt/homebrew/bin/yarn
    npm: 8.15.0 - /opt/homebrew/bin/npm
    Watchman: 2022.07.04.00 - /opt/homebrew/bin/watchman
  Managers:
    CocoaPods: 1.11.3 - /Users/sontb/.rvm/gems/ruby-2.7.5/bin/pod
  SDKs:
    iOS SDK:
      Platforms: DriverKit 21.4, iOS 15.5, macOS 12.3, tvOS 15.4, watchOS 8.5
    Android SDK:
      API Levels: 23, 28, 29, 30, 31, 32
      Build Tools: 28.0.3, 29.0.2, 29.0.3, 30.0.2, 30.0.3, 31.0.0
      System Images: android-31 | Android TV ARM 64 v8a, android-31 | Google APIs ARM 64 v8a
      Android NDK: Not Found
  IDEs:
    Android Studio: 2021.2 AI-212.5712.43.2112.8609683
    Xcode: 13.4.1/13F100 - /usr/bin/xcodebuild
  Languages:
    Java: 11.0.14.1 - /usr/bin/javac
  npmPackages:
    @react-native-community/cli: Not Found
    react: 18.1.0 => 18.1.0 
    react-native: 0.70.1 => 0.70.1 
    react-native-macos: Not Found
  npmGlobalPackages:
    *react-native*: Not Found
```

datetimepicker version: 6.4.0

**Comment by kalyantm** [comment]
I'm also facing the same issue!

**Comment by vonovak** [comment]
:tada: This issue has been resolved in version 6.4.1 :tada:

If this package helps you, consider [sponsoring us](https://github.com/sponsors/react-native-datetimepicker)! :rocket:
